qa_forum/static/js/main.js

```diff
@@ -0,0 +1,39 @@
+$(document).ready(function(){
+
+    $('.comment_submit').on('click', function() {
+        var raiser = $(this).prev();
+        var comment = $(this).parent().prev().children();
+        var pointTo = $(this).val();
+        if (raiser.val() && comment.val()) {
+            send_comment(raiser, comment, pointTo);
+        } else {
+            alert("Please fill in both your name and the comment")
+        }
+        clear(raiser, comment);
+    });
+
+    function clear(raiser, comment){
+        raiser.val("");
+        comment.val("");
+    }
+
+    function send_comment(raiser, comment, pointTo) {
+        var data = {
+            "raiser": raiser.val(),
+            "comment": comment.val(),
+            "pointQ": pointTo
+        };
+        $.ajax({
+            url: '/answer_form/',
+            type: 'POST',
+            dataType: 'json',
+            data: JSON.stringify(data),
+            success: function() {
+                window.location.href = "/";
```

Comment:
@chuckcwh success callback should return the answer created? You can probably use the data received to create a new DOM node and push it to the answers. Otherwise, there's no point doing AJAX.